rllib/evaluation/rollout_worker.py

```diff
@@ -420,7 +420,7 @@ def make_env(vector_index):
         if (ray.is_initialized()
                 and ray.worker._mode() != ray.worker.LOCAL_MODE):
             # Check available number of GPUs
-            if not ray.get_gpu_ids(as_str=True):
+            if not ray.get_gpu_ids():
```

Comment:
I believe as_str=True is correct to avoid crashes in certain cases, reading get_gpu_ids(). Please avoid making unrelated changes in PRs.

Reply:
Done. It did fail in one of my runs, though. Shouldn't matter here anyways, we just check for an empty list.